starter-core/src/main/java/io/micronaut/starter/feature/awslambdacustomruntime/AwsLambdaCustomRuntime.java

```diff
@@ -51,11 +51,11 @@ public class AwsLambdaCustomRuntime implements FunctionFeature, ApplicationFeatu
     public static final String FEATURE_NAME_AWS_LAMBDA_CUSTOM_RUNTIME = "aws-lambda-custom-runtime";
 
     private final Provider<AwsLambda> awsLambda;
-    private final HttpClient httpClient;
+    private final HttpClientJdk httpClientJdk;
 
-    public AwsLambdaCustomRuntime(Provider<AwsLambda> awsLambda, HttpClient httpClient) {
```

Comment:
we should deprecate the old constructor

Reply:
resolved via https://github.com/micronaut-projects/micronaut-starter/pull/2272/commits/a66e38845727bcae3f66de931512b8cf98a85c85 and https://github.com/micronaut-projects/micronaut-starter/pull/2272/commits/07b6e0db5f48cb220467f88bad443b0ff2081e5e